test/WebSites/FormatterWebSite/Models/User.cs

```diff
@@ -2,23 +2,30 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
 
 namespace FormatterWebSite
 {
+    [DataContract]
```

Comment:
The tests using this model failed due to the new `Required` attribute validation...so fixing this..